.github/workflows/publish-android.yml

```diff
@@ -71,8 +71,8 @@ jobs:
           fi
 
       - name: Gradle Publish Android Package
-        uses: gradle/gradle-build-action@v2
+        uses: gradle/actions/setup-gradle@v3
         with:
-          arguments: publishAndroidReleasePublicationToGithubPackagesRepository ${{ env.PUB_MODE }}
+          arguments: publishAndroidReleasePublicationToGithubPackagesRepository -pandroid=true ${{ env.PUB_MODE }}
```

Comment:
Careful with the `-pandroid=true`, it should be `-Pandroid=true`, otherwise it won't work.

Reply:
Fixed